setup.py

```diff
@@ -39,10 +39,6 @@ def rel(*xs):
         raise RuntimeError("Version marker not found.")
 
 
-dependencies = [
-    "prometheus-client>=0.2",
```

Comment:
This would technically be a breaking change to the build process for people who use Dramatiq, so I'm not sure what to do about it. Is there a way to specify negative dependencies in `setup.py`? Eg., `pip install dramatiq[no-prometheus]`. If so, that's the way we should go.

Reply:
I am not sure that there is a thing such as negative dependencies. I think this is covered by [PEP771](https://peps.python.org/pep-0771/) but this is still a draft at the moment.

Removing `prometheus-client` is indeed a breaking change, and I am not sure how this could be otherwise. Usually such changes come with deprecation warnings, but in the case of dependencies, I don't know how to raise a deprecation warning for users that use the middleware but have not explicitly added its extra.

The only remaining option I see is good communication on the changelog.